csm/conf/csm_cleanup.py

```diff
@@ -79,38 +86,64 @@ def remove_old_indexes(es, arg_d, arg_n, arg_e):
+        Log.debug(f"Removing data for old index:{index} for {no_of_days} days.")
+        es.remove_old_data_from_indexes(no_of_days, host_port, [index], field)
+    remove_old_indexes(es, no_of_days, host_port, args.emulate)
+
+def get_du_data():
+    cmd = "sudo du -BM /var/log/elasticsearch/"
+    sp_es = SimpleProcess(cmd)
+    Log.debug(f"Running {cmd}")
+    res = sp_es.run()
+    Log.debug(f"Resulted: {res}")
+    res = int(res[0].decode("utf-8").split('\t')[0].split('M')[0])
+    return res
+
+def get_df_data():
+    cmd = " df -BM /var/log"
+    sp_es = SimpleProcess(cmd)
+    Log.debug(f"Running {cmd}")
+    cmd_data = sp_es.run()
+    Log.debug(f"Resulted: {cmd_data}")
+    storage_info = cmd_data[0].decode('utf-8').split('\n').pop(1).split(' ')
+    result = [ele for ele in storage_info if len(ele)>0]
+    return int(result[1].split('M')[0]),  int(result[4].split('%')[0])
+
+def process_es_cleanup(args):
     # Pass arguments to worker function
     # remove data older than given number of days
     es = esCleanup(const.CSM_CLEANUP_LOG_FILE, const.CSM_LOG_PATH)
-    es.remove_old_data_from_indexes(args.d, args.n, args.i, args.f)
-
-def add_stats_subcommand(main_parser):
-    subparsers = main_parser.add_parser("stats", help='cleanup of stats log')
-    subparsers.set_defaults(func=process_stats)
-    subparsers.add_argument("-d", type=int, default=90,
-            help="days to keep data")
-    subparsers.add_argument("-n", type=str, default="localhost:9200",
-            help="address:port of elasticsearch service")
-    subparsers.add_argument("-e", action='store_true',
-            help="emulate, do not really delete indexes")
-
-def add_auditlog_subcommand(main_parser):
-    subparsers = main_parser.add_parser("auditlogs", help='cleanup of audit log')
-    subparsers.set_defaults(func=process_auditlogs)
-    subparsers.add_argument("-d", type=int, default=90,
-            help="days to keep data")
-    subparsers.add_argument("-n", type=str, default="localhost:9200",
-            help="address:port of elasticsearch service")
-    subparsers.add_argument("-f", type=str, default="timestamp",
-            help="field of index of elasticsearch service")
-    subparsers.add_argument("-i", nargs='+', default=[],
-            help="index of elasticsearch")
+    days_to_keep_data = int(args.days_to_keep_data)
+    clean_indexes(es, days_to_keep_data, args.host_port)
+    var_log_storage, var_log_usage_percent = get_df_data() #get current /var/log storage
+
+    #calculate es_db_capp Eg. var_log_storage=8000MB es_storage_cap_percent=30%
+    es_db_capping = (var_log_storage * int(args.es_storage_cap_percent)) / 100
+
+    while var_log_usage_percent > int(args.var_log_cap_percent):
+        days_to_keep_data = days_to_keep_data-1
+        # Break if current ES storage is less than ES capping OR
+        # Break if no of days is less than or equal 2
+        if (get_du_data() <= es_db_capping) or (days_to_keep_data<=2):
```

Comment:
Code does not having consistency. At places space is given `get_du_data() <= es_db_capping` and in the same line no spacing `days_to_keep_data<=2`
Make sure that all the coding standards are followed.

Reply:
Comment addressed. Please review